src/Console/CodegenCommand.php

```diff
@@ -5,24 +5,39 @@
 use Spawnia\Sailor\Codegen\Generator;
 use Spawnia\Sailor\Codegen\Writer;
 use Spawnia\Sailor\Configuration;
+use SplFileInfo;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class CodegenCommand extends Command
 {
+    private const OPTION_CONFIGURATION = 'configuration';
```

Comment:
```suggestion
    private const OPTION_CONFIG = 'config';
```

I am generally for descriptive and unabbreviated names, but I have never seen `--configuration` spelled out as a CLI option.